crates/cli/src/cli/mod.rs

```diff
@@ -125,71 +125,71 @@ enum Command {
 
 #[derive(Parser, PartialEq, Clone, Debug)]
 pub struct StartSimnet {
-    /// Path to the manifest
+    /// Path to the manifest that contains the Simnet configuration (eg. surfpool start --manifest-file-path ./Surfpool.toml)
```

Comment:
This is a confusing field. We actually aren't using the file we point to for anything, just using the directory it's in. So currently, we have no concept of a "Surfpool.toml". 

I suppose we will one day. For now, maybe we could do this? Thoughts, @lgalabru 

```suggestion
    /// Path to the runbook manifest, used to locate the root of the project (eg. surfpool start --manifest-file-path ./txtx.toml)
```